### DISCUSSION TITLE
graph.pb completely destroys the molecular dynamics structure under 300K

### DISCUSSION THREAD

**Discussion by 919607** [deepmodeling/deepmd-kit#2342] [Q&A]
Dear teachers, may I ask why my lattice structure completely falls apart when I run an NPT molecular [dynamics]([url](url)) simulation at 300K using my trained graph.pb?

Before developing this potential function, I created a rough model of the potential function, which had slightly lower accuracy but did not result in the complete breakdown of the lattice structure as seen in my current model. That's why I decided to train a new, more accurate potential function. The only difference between the rough model and the current model is the initial training set. The rough model was trained on the SCF output of VASP, while the current model was trained on 10 steps of AIMD output from VASP.

The AIMD data provides more information than the SCF data, so the current potential is twice as accurate as the rough model, based on its performance. However, the current graph.pb is unable to produce satisfactory NPT results, even though it has higher accuracy compared to the rough model.

**Comment by njzjz** [comment]
For accuracy issue please check the documentation: https://docs.deepmodeling.com/projects/deepmd/en/v2.2.0.b0/troubleshooting/precision.html
